RTC-Lab Seville SPAIN

alinaresb opened the pull request:
I have added my group to the list and three journal papers on visual event-based processing.
Thanks.

guillermogb commented on README.md:
```diff
@@ -226,6 +226,9 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2019. [PDF](https://arxiv.org
 - <a name="Gehrig18eccv"></a>Gehrig, D., Rebecq, H., Gallego, G., Scaramuzza, D.,  
 *[Asynchronous, Photometric Feature Tracking using Events and Frames](http://rpg.ifi.uzh.ch/docs/ECCV18_Gehrig.pdf)*,  
 European Conf. Computer Vision (ECCV), 2018. [Poster](http://rpg.ifi.uzh.ch/docs/ECCV18_Gehrig_poster.pdf), [YouTube](https://youtu.be/A7UfeUnG6c4), [Oral presentation](https://youtu.be/7EvY8SxdLl8), [Tracking analysis code](https://github.com/uzh-rpg/rpg_feature_tracking_analysis)
+- <a name="LinaresBarrancoA18entropy"></a>Linares-Barranco, A., Liu, H., Rios-Navarro, A., Gomez-Rodriguez, F., Moeys, D., Delbruck, T.
```
Need to leave two blank spaces at the end of the line so that there is a line break on github.
We want the author names, paper title and publication venue in separate lines. 
Can you fix this by looking at the preview?

guillermogb commented on README.md:
```diff
@@ -254,6 +257,9 @@ IS&T International Symposium on Electronic Imaging 2019. Computational Imaging X
 - <a name="Alzugaray193dv"></a>Alzugaray, I., Chli, M.,  
 *[Asynchronous Multi-Hypothesis Tracking of Features with Event Cameras](https://doi.org/10.3929/ethz-b-000360434)*,  
 Int. Conf. 3D Vision (3DV), 2019. [PDF](https://doi.org/10.3929/ethz-b-000360434), [YouTube](https://youtu.be/eguV_AIbteU)
+- <a name="LinaresBarrancoA19access"></a>Linares-Barranco, A., Perez-Pena, F., Moeys, D., Gomez-Rodriguez, F., Jimenez-Moreno, G., Delbruck, T. 
```
Same thing here. Need two white spaces at the end of the line to create a line break.

guillermogb commented on README.md:
```diff
@@ -254,6 +257,9 @@ IS&T International Symposium on Electronic Imaging 2019. Computational Imaging X
 Int. Conf. 3D Vision (3DV), 2019. [PDF](https://doi.org/10.3929/ethz-b-000360434), [YouTube](https://youtu.be/eguV_AIbteU)
+- <a name="LinaresBarrancoA19access"></a>Linares-Barranco, A., Perez-Pena, F., Moeys, D., Gomez-Rodriguez, F., Jimenez-Moreno, G., Delbruck, T. 
+*[Low Latency Event-based Filtering and Feature Extraction for Dynamic Vision Sensors in Real-Time FPGA Applications](https://doi.org/10.1109/ACCESS.2019.2941282)*,  
+IEEE-Acess 2019, early access.
```
No need for "early access", thanks.

guillermogb commented on README.md:
```diff
@@ -820,6 +825,9 @@ Robotics: Science and Systems XIV (RSS), 2019. [PDF](https://www.research-collec
 - <a name="Chen19fnins"></a>Chen, G., Chen, J., Lienen, M., Conradt, J., Roehrbein, F., Knoll, A.C.,  
 *[FLGR: Fixed Length Gists Representation Learning for RNN-HMM Hybrid-Based Neuromorphic Continuous Gesture Recognition](https://dx.doi.org/10.3389%2Ffnins.2019.00073)*,  
 Front. Neurosci., 2019, 13:73.
+- <a name="Tapiador18tbiocas"></a>Tapiador-Morales, R., Linares-Barranco, A., Jimenez-Fernandez, A., Jimenez-Moreno, G. 
```
Same here: need two white spaces at the end.

guillermogb commented on README.md:
```diff
@@ -820,6 +825,9 @@ Robotics: Science and Systems XIV (RSS), 2019. [PDF](https://www.research-collec
 Front. Neurosci., 2019, 13:73.
+- <a name="Tapiador18tbiocas"></a>Tapiador-Morales, R., Linares-Barranco, A., Jimenez-Fernandez, A., Jimenez-Moreno, G. 
+*[Neuromorphic LIF Row-by-Row Multiconvolution Processor for FPGA](https://doi.org/10.1109/TBCAS.2018.2880012)*,  
+IEEE Transactions on Biomedical Circuits and Systems, 2019, vol. 13, issue 1.
```
Can you abbreviate the name of the journal as in IEEE convention?
http://technicalghostwriters.com/wp-content/uploads/2012/02/IEEE-Abbreviations-for-Transactions-Journals-Letters.pdf
thanks

guillermogb commented on README.md:
```diff
@@ -1313,6 +1321,7 @@ MSc. Thesis, Air Force Institute of Technology, USA, 2019. [PDF](https://apps.dt
 - [Western Sydney University’s International Centre for Neuromorphic Systems (ICNS)](https://www.westernsydney.edu.au/icns), Australia.
 - [Perception and Robotics Group](http://prg.cs.umd.edu/) at University of Maryland (UMD).
 - [Intel Labs](https://www.intel.com/content/www/us/en/research/overview.html), [Mike Davies](https://newsroom.intel.com/news/bobbleheads-tool-test-neuromorphic-mind/#gs.nndw9d) (Intel’s neuromorphic computing program leader).
+- [RTC-lab Sevilla](http://www.rtc.us.es/), University of Seville, SPAIN
```
Perhaps you want to add "Real Time Communication Laboratory" to show what RTC means?

alinaresb commented:
Thanks,
I'm fixing and pushing it again.
Regards,
 Alejandro.

--

Prof. Alejandro Linares Barranco

Director departamento de Arquitectura y Tecnología de Computadores
[https://sic.us.es/sites/default/files/servicios/correo/Archivos/us_logo.jpg]   [http://www.atc.us.es/images/ATCLOGO90.png]
www.us.es <http://www.us.es/>
Por favor considere el medio ambiente antes de imprimir este correo electrónico.
-
Este correo electrónico y, en su caso, cualquier fichero anexo al mismo, contiene información de carácter confidencial exclusivamente dirigida a su destinatario o destinatarios. Si no es Vd. el destinatario del mensaje, le ruego lo destruya sin hacer copia digital o física, comunicando al emisor por esta misma vía la recepción del presente mensaje. Gracias

________________________________
De: Guillermo Gallego <notifications@github.com>
Enviado: lunes, 16 de septiembre de 2019 17:41
Para: uzh-rpg/event-based_vision_resources <event-based_vision_resources@noreply.github.com>
Cc: Alejandro Linares-Barranco <alinares@atc.us.es>; Author <author@noreply.github.com>
Asunto: Re: [uzh-rpg/event-based_vision_resources] RTC-Lab Seville SPAIN (#78)


@guillermogb requested changes on this pull request.

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324744846>:

> @@ -226,6 +226,9 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2019. [PDF](https://arxiv.org

 - <a name="Gehrig18eccv"></a>Gehrig, D., Rebecq, H., Gallego, G., Scaramuzza, D.,

 *[Asynchronous, Photometric Feature Tracking using Events and Frames](http://rpg.ifi.uzh.ch/docs/ECCV18_Gehrig.pdf)*,

 European Conf. Computer Vision (ECCV), 2018. [Poster](http://rpg.ifi.uzh.ch/docs/ECCV18_Gehrig_poster.pdf), [YouTube](https://youtu.be/A7UfeUnG6c4), [Oral presentation](https://youtu.be/7EvY8SxdLl8), [Tracking analysis code](https://github.com/uzh-rpg/rpg_feature_tracking_analysis)

+- <a name="LinaresBarrancoA18entropy"></a>Linares-Barranco, A., Liu, H., Rios-Navarro, A., Gomez-Rodriguez, F., Moeys, D., Delbruck, T.



Need to leave two blank spaces at the end of the line so that there is a line break on github.
We want the author names, paper title and publication venue in separate lines.
Can you fix this by looking at the preview?

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324745217>:

> @@ -254,6 +257,9 @@ IS&T International Symposium on Electronic Imaging 2019. Computational Imaging X

 - <a name="Alzugaray193dv"></a>Alzugaray, I., Chli, M.,

 *[Asynchronous Multi-Hypothesis Tracking of Features with Event Cameras](https://doi.org/10.3929/ethz-b-000360434)*,

 Int. Conf. 3D Vision (3DV), 2019. [PDF](https://doi.org/10.3929/ethz-b-000360434), [YouTube](https://youtu.be/eguV_AIbteU)

+- <a name="LinaresBarrancoA19access"></a>Linares-Barranco, A., Perez-Pena, F., Moeys, D., Gomez-Rodriguez, F., Jimenez-Moreno, G., Delbruck, T.



Same thing here. Need two white spaces at the end of the line to create a line break.

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324745378>:

> @@ -254,6 +257,9 @@ IS&T International Symposium on Electronic Imaging 2019. Computational Imaging X

 - <a name="Alzugaray193dv"></a>Alzugaray, I., Chli, M.,

 *[Asynchronous Multi-Hypothesis Tracking of Features with Event Cameras](https://doi.org/10.3929/ethz-b-000360434)*,

 Int. Conf. 3D Vision (3DV), 2019. [PDF](https://doi.org/10.3929/ethz-b-000360434), [YouTube](https://youtu.be/eguV_AIbteU)

+- <a name="LinaresBarrancoA19access"></a>Linares-Barranco, A., Perez-Pena, F., Moeys, D., Gomez-Rodriguez, F., Jimenez-Moreno, G., Delbruck, T.

+*[Low Latency Event-based Filtering and Feature Extraction for Dynamic Vision Sensors in Real-Time FPGA Applications](https://doi.org/10.1109/ACCESS.2019.2941282)*,

+IEEE-Acess 2019, early access.



No need for "early access", thanks.

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324745879>:

> @@ -820,6 +825,9 @@ Robotics: Science and Systems XIV (RSS), 2019. [PDF](https://www.research-collec

 - <a name="Chen19fnins"></a>Chen, G., Chen, J., Lienen, M., Conradt, J., Roehrbein, F., Knoll, A.C.,

 *[FLGR: Fixed Length Gists Representation Learning for RNN-HMM Hybrid-Based Neuromorphic Continuous Gesture Recognition](https://dx.doi.org/10.3389%2Ffnins.2019.00073)*,

 Front. Neurosci., 2019, 13:73.

+- <a name="Tapiador18tbiocas"></a>Tapiador-Morales, R., Linares-Barranco, A., Jimenez-Fernandez, A., Jimenez-Moreno, G.



Same here: need two white spaces at the end.

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324746058>:

> @@ -820,6 +825,9 @@ Robotics: Science and Systems XIV (RSS), 2019. [PDF](https://www.research-collec

 - <a name="Chen19fnins"></a>Chen, G., Chen, J., Lienen, M., Conradt, J., Roehrbein, F., Knoll, A.C.,

 *[FLGR: Fixed Length Gists Representation Learning for RNN-HMM Hybrid-Based Neuromorphic Continuous Gesture Recognition](https://dx.doi.org/10.3389%2Ffnins.2019.00073)*,

 Front. Neurosci., 2019, 13:73.

+- <a name="Tapiador18tbiocas"></a>Tapiador-Morales, R., Linares-Barranco, A., Jimenez-Fernandez, A., Jimenez-Moreno, G.

+*[Neuromorphic LIF Row-by-Row Multiconvolution Processor for FPGA](https://doi.org/10.1109/TBCAS.2018.2880012)*,

+IEEE Transactions on Biomedical Circuits and Systems, 2019, vol. 13, issue 1.



Can you abbreviate the name of the journal as in IEEE convention?
http://technicalghostwriters.com/wp-content/uploads/2012/02/IEEE-Abbreviations-for-Transactions-Journals-Letters.pdf
thanks

________________________________

In README.md<https://github.com/uzh-rpg/event-based_vision_resources/pull/78#discussion_r324746500>:

> @@ -1313,6 +1321,7 @@ MSc. Thesis, Air Force Institute of Technology, USA, 2019. [PDF](https://apps.dt

 - [Western Sydney University’s International Centre for Neuromorphic Systems (ICNS)](https://www.westernsydney.edu.au/icns), Australia.

 - [Perception and Robotics Group](http://prg.cs.umd.edu/) at University of Maryland (UMD).

 - [Intel Labs](https://www.intel.com/content/www/us/en/research/overview.html), [Mike Davies](https://newsroom.intel.com/news/bobbleheads-tool-test-neuromorphic-mind/#gs.nndw9d) (Intel’s neuromorphic computing program leader).

+- [RTC-lab Sevilla](http://www.rtc.us.es/), University of Seville, SPAIN



Perhaps you want to add "Real Time Communication Laboratory" to show what RTC means?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/uzh-rpg/event-based_vision_resources/pull/78?email_source=notifications&email_token=ADWOVSSP53SRYVNVWSU6YZ3QJ6SJHA5CNFSM4IXD2YZ2YY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCE2YHZI#pullrequestreview-288719845>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ADWOVSXI5VQMWJ25DN3VGBTQJ6SJHANCNFSM4IXD2YZQ>.

alinaresb commented:
I have commited the changes you addressed to RTC-research-group:master. Can you see them?

guillermogb commented:
Yes, I can see the changes, thanks. But you closed the pull request...

guillermogb commented:
thanks
